cmd/common.go

```diff
@@ -27,6 +27,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+var kubeToCISMap = map[string]string{
+	"1.11": "cis-1.3.0",
+	"1.12": "cis-1.3.0",
+	"1.13": "cis-1.4.1",
+	"1.14": "cis-1.4.1",
+	"1.15": "cis-1.4.1",
+	"1.16": "cis-1.4.1",
+	"1.17": "cis-1.4.1",
+}
```

Comment:
Let's keep this future-resistant. If we kept the previous version of `decrementVersion` which decrements the kubernetes version, we could decrement until we find a version that has an entry in this table. So, something like

```
cisVersion, ok = kubeToCISmap[kubeVersion]
if !ok {
    for ; kubeVersion = decrementVersion(kubeVersion); (kubeVersion != 1.11 && !ok) { 
        cisVersion, ok = kubeToCISmap[kubeVersion]
   }
}

if !ok {
   we didn't find a version so error
}
```

Reply:
I used your idea as a starting point and made changes. wdyt?